lib/wda/webdriveragent.js

```diff
@@ -111,12 +116,13 @@ class WebDriverAgent {
 
     await this.xcodebuild.init(this.noSessionProxy);
 
-    if (this.prebuildWDA) {
-      await this.xcodebuild.prebuild();
-    }
-
     // start the xcodebuild process
-    return await this.xcodebuild.start();
+    return await retryInterval(this.realDevice ? 1 : 3, 3000 + randomInt(1, 2000), async () => {
```

Comment:
usually everything tries to start at the same time while parallel tests are running. This random period should reduce the possibility for different xcodebuild instances to clash into conflicting situation because of simultaneous resource locking.